canvas_modules/common-canvas/src/palette/palette-flyout-content.jsx

```diff
@@ -127,7 +108,6 @@ class PaletteFlyoutContent extends React.Component {
 				canvasController={this.props.canvasController}
 				isPaletteOpen={this.props.isPaletteOpen}
 				isEditingEnabled={this.props.isEditingEnabled}
-				// isShowRanking // Uncomment this to show ranking for debuggig ranking algorithm
```

Comment:
In the earlier comment I asked you to put this line back in and your response says "Fixed" -- but it is not there?! What happened? I don't understand how you can say it is Fixed when it isn't.

Reply:
Did put it back !!